### ISSUE TITLE
Using with create-react-app

### ISSUE THREAD

**Issue by Dagur** [vanilla-extract-css/vanilla-extract#535]
Hi

I just created a new create-app-app app (with typescript) and want to use vanilla-extract. There isn't much info about how to do that except https://tsh.io/blog/vanilla-extract-library/ and that doesn't work with the latest CRA. 

Can you add something to the documentation on how to do this? I'm sure a lot of people would appreciate it.

**Comment by calebpdx** [comment]
The official documentation shows how to install it with Next.js:
https://vanilla-extract.style/documentation/setup/#nextjs

You can bypass the babel plugin if you want to run it with SWC. However, there is a bug somewhere in Vanilla Extract that causes a PostCSS error in Next beyond Next@12.0.2, waiting for some help to get it all cleared up!

**Comment by Dagur** [comment]
The Next.js instructions can't be used with CRA.

Anyway, since this was a new project I opted for Vite instead. That worked without issues.

**Comment by Dremora** [comment]
You can't use vanilla-extract with plain create-react-app, however, you can use it with [Craco](https://github.com/gsoft-inc/craco). It allows you to override webpack config, and vanilla-extract webpack plugin seems to run without issues there.

**Comment by Dagur** [comment]
I tried that first using instructions that I found on Google but there's version conflicts with the latest version of create-react-app

**Comment by Spaubleit** [comment]
If you use yarn, you no longer need craco. You can patch react-scripts config using yarn patch command.

**Comment by mattcompiles** [comment]
Not sure there's anything to resolve here outside of providing some docs on how to setup VE with create-react-app. Happy to accept a PR adding setup docs for create-react-app if someone is keen to contribute.

**Comment by rothsandro** [comment]
I installed VE in a CRA@5.0.1 app and configured the webpack and babel plugin with Craco. The dev server and the tests are working fine but the production build fails because VE imports a file from node_modules:

```
You attempted to import node_modules/@vanilla-extract/webpack-plugin/extracted.js 
which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
```

I configured a path to this folder in tsconfig using react-app-alias and it works now. Is this the recommended solution or is there a better way to use VE with CRA?
